.github/workflows/task-test.yml

```diff
@@ -182,6 +182,7 @@ jobs:
           SAN: ${{ inputs.san }}
           REDIS_VER: ${{ inputs.get-redis }}
           ENABLE_ASSERT: 1
+          SVS_SHARED_LIB: 1
```

Comment:
I'm not comfortable with the fact that the CI never runs with the default value (SVS_SHARED_LIB=0)

Reply:
I am not either, but believe it is the best way to go as long as we have to choose one, and given that we have that CI does run with SVS_SHARED_LIB=0 for non x86+Linux platforms